src/common/hooks/transaction/use-transaction-error.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  useTransactionContractInterface,
+  useTransactionRequest,
+} from '@common/hooks/transaction/use-transaction';
+import { useRecoilValue } from 'recoil';
```

Comment:
Should we organize imports so external libraries are listed first?

Reply:
yeah it'd be nice to have an eslint rule, @kyranjamie I know you mentioned this before